.github/workflows/rollback.yml

```
@@ -1,389 +1,389 @@
+          else
+            echo "- :x: Rollback: ${{ needs.rollback.result }}" >> $GITHUB_STEP_SUMMARY
+          fi
+
+          if [ "${{ needs.verify.result }}" == "success" ]; then
+            echo "- :white_check_mark: Verification: Passed" >> $GITHUB_STEP_SUMMARY
+          else
+            echo "- :x: Verification: ${{ needs.verify.result }}" >> $GITHUB_STEP_SUMMARY
+          fi
+
+      - name: Send Slack notification
+        if: ${{ secrets.SLACK_WEBHOOK_URL != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)